test/extensions.test.ts

```diff
@@ -29,6 +29,66 @@ const tx = {
     }
 };
 
+const burnTxData = 'ughWYOfsCrMICkBDNTI1MUI4Rjg4QUYyMzZDQzVFNzVDQjc3NzhEMjFDRjFDQzEzNTZEOTA2QTVDNjVGMUJEQ0Q1Mzk2MTVFRjMxEJeeBCKZBhqWBlt7Im1zZ19pbmRleCI6MCwic3VjY2VzcyI6dHJ1ZSwibG9nIjoiIiwiZXZlbnRzIjpbeyJ0eXBlIjoiYnVybiIsImF0dHJpYnV0ZXMiOlt7ImtleSI6ImV0aGVyZXVtX2NoYWluX2lkIiwidmFsdWUiOiIzIn0seyJrZXkiOiJ0b2tlbl9jb250cmFjdCIsInZhbHVlIjoiMHg2ODJjMkFFNDA1M0VhYzY0Y0YxQkFhQTA0QzczOTcwM0RjMDQzRjBBIn0seyJrZXkiOiJjb3Ntb3Nfc2VuZGVyIiwidmFsdWUiOiJjb3Ntb3MxcXdudzJyOWFrNzk1MzZjNGRxdHJ0azJwbDJubHpwcWg3NjNybHMifSx7ImtleSI6ImV0aGVyZXVtX3JlY2VpdmVyIiwidmFsdWUiOiIweDdCOTVCNkVDN0ViRDczNTcyMjk4Y0VmMzJCYjU0RkE0MDgyMDczNTkifSx7ImtleSI6ImFtb3VudCIsInZhbHVlIjoiMXN0YWtlIn1dfSx7InR5cGUiOiJtZXNzYWdlIiwiYXR0cmlidXRlcyI6W3sia2V5Ijoic2VuZGVyIiwidmFsdWUiOiJjb3Ntb3MxcXdudzJyOWFrNzk1MzZjNGRxdHJ0azJwbDJubHpwcWg3NjNybHMifSx7ImtleSI6Im1vZHVsZSIsInZhbHVlIjoiZXRoYnJpZGdlIn0seyJrZXkiOiJzZW5kZXIiLCJ2YWx1ZSI6ImNvc21vczFxd253MnI5YWs3OTUzNmM0ZHF0cnRrMnBsMm5senBxaDc2M3JscyJ9LHsia2V5IjoiYWN0aW9uIiwidmFsdWUiOiJidXJuIn1dfSx7InR5cGUiOiJ0cmFuc2ZlciIsImF0dHJpYnV0ZXMiOlt7ImtleSI6InJlY2lwaWVudCIsInZhbHVlIjoiY29zbW9zMWwzZGZ0ZjQ5OXU0Z3ZkZXV1emRsMnBndjRmMHhkdG51ZW5zNXd2In0seyJrZXkiOiJhbW91bnQiLCJ2YWx1ZSI6IjFzdGFrZSJ9XX1dfV0qzgHMASgoFqkKVID+h3cIAxIUaCwq5AU+rGTPG6qgTHOXA9wEPwoaFAOm5Qy9t4tI6xVoFjXZQfqn8QQXIhR7lbbsfr1zVyKYzvMrtU+kCCBzWSoKCgVzdGFrZRIBMRIEEMCaDBpqCibrWumHIQLSd46NU8TNPq5KoiQ5gsf1NOEc570jHMOI5iOsRLIc1BJAVb3H8EYCymIjOpGOXRQc/yUpdU0Uneyin+xkE2ZSdhxabP/8VVHl+wq5bmcPKH8xX2DqJDEFag9pu/rZfwSRvA=='
```

Comment:
Was this derived from encoding `burnTx` using this library, or encoding with go-amino (or obtaining it from a source that did so)? If the former, it's always a good idea to make sure that the canonical Amino encoding matches this.

Reply:
Was derived by encoding `burnTx` using this library. Looks like this is probably contributing to the tests failing. Grabbing from the source sounds preferable, as we'll know the encoded data is from a properly structured tx.